tests/console/commands/test_export.py

```diff
@@ -90,9 +92,11 @@ def test_export_exports_requirements_txt_file_locks_if_no_lock_file(app, repo):
 
     assert app.poetry.locker.lock.exists()
 
-    expected = """\
-foo==1.0.0
-"""
+    expected = dedent(
```

Comment:
sorry for this side fix, but I really hate when code alignment is broken, especially in unit test where perf isn't that key...